test/modules/jwt-encryption-service/JwtEncryptionService.test.ts

```diff
@@ -0,0 +1,29 @@
+import * as assert from 'assert';
+
+import JwtEncryptionService from 'app/modules/jwt-encryption-service/JwtEncryptionService';
+
+
+describe('JwtEncryptionService', () => {
+
+    const service: JwtEncryptionService = new JwtEncryptionService({
+        oath_scope_prefix: '',
+        chsUrl: '',
+        accountClientId: '',
+        accountRequestKey: 'pXf+qkU6P6SAoY2lKW0FtKMS4PylaNA3pY2sUQxNFDk=',
+        accountUrl: ''
+    });
+
+    it('should generate random nonce value in base64', () => {
```

Comment:
Not sure what that test does? It does not test randomness (because it is invoked only once). It does not test base 64 encoding really (because broken base 64 encoded string can still conform regex).

Reply:
How could this be tested? Running it multiple times and checking if values are different? two random values could happen to be equal (even tho chances are slim) and fail the test.

Reply:
Can randomness be increased beyond 5 bytes? That touches on question that Rhys had.

Reply:
I increased it to 10, and it seems to work fine. Cant find documentation on any requirements.

Reply:
Seems like nounce does not have to be too long, sometimes 32 bits (4 bytes) is enough and to be fairs as long as these are cryptographically random chance of collision is very small. Lets keep it at current length and lets generate it twice to make sure they are not the same.